lora/lorawan/region/au915.go

```diff
@@ -0,0 +1,27 @@
+package region
+
+import "tinygo.org/x/drivers/lora"
+
+type RegionSetAU915 struct {
+	defaultTxPower        int
+	defaultPreambleLength int
+}
+
+func NewRegionSetAU915() RegionSetAU915 {
+	r := RegionSetAU915{}
```

Comment:
This could be simplified to:

```go
return RegionSetAU915{defaultTxPower, 20, defaultPreambleLength: 8}
```

Also, I do not see where either `defaultTxPower` or `defaultPreambleLength` are actually used?

Reply:
Thanks. 
I'll fix this part of code and push an update tonight.